src/pocketmine/entity/Living.php

```diff
@@ -233,15 +233,18 @@ public function hasEffect(int $effectId) : bool{
 	 * If a weaker or equal-strength effect is already applied but has a shorter duration, it will be replaced.
 	 *
 	 * @param Effect $effect
+	 *
+ 	 * Returns whether the effect has been successfully applied.
```

Comment:
I think @SOF3 meant something more like `@return bool Whether the effect has been successfully applied`

Reply:
ah yeah he probably did